### PR TITLE
update vizier keywords lists

### PR DESCRIPTION
**Pull request by ManonMarchand**
Hi all :slightly_smiling_face: 

This PR updates the keywords list for VizieR. 

It modifies the json files and also includes a tiny script to renew them more often. I can remove this from the PR if this is not a good place for such a script.

As a result, there will be no errors for the new missions and objects (like "Exoplanets" or "Gravitational_wave"). 

New objects and missions are pretty rare, but is there a way to implement a renewal every few weeks/months? Maybe linked to the releases of astroquery?

### CODE REVIEW

**Comment by keflavich** [comment]
I like the idea of regularly updating.  Maybe we can ask the CDS team to push a notification whenever they update the keywords?

File: inverse_dict.json
Comment (bsipocz) [inline inverse_dict.json:1]: These should be in the vizier/data directory, not at top level

**Comment by bsipocz** [comment]
I wonder whether it even makes sense to keep these files around, and rather just have a method or function that runs the TAP query? That's what we do in other modules, and given these are not really big data in any sense, it shouldn't be controversial to have the up-to-date version from the server rather than a bundled version.

File: inverse_dict.json
Comment (ManonMarchand) [reply inverse_dict.json:1]: OMG sorry!

**Comment by ManonMarchand** [comment]
Hello, thanks for the feedback :slightly_smiling_face: 

For now, we can already update the files with this PR. 

I asked the team, and the keywords list changes less than two times a year, and we'd rather avoid an additional query on our side for something this static. Moreover, there is an ongoing change: we'll soon follow the UAT keywords, meaning that the current list will become legacy (still supported but will never change again). 

So the solution I'm offering for now is that I'll manually do the update when needed, and the keywords support will be refactored when the UAT changes comes to production. 

Do you think it makes sense?

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
@ManonMarchand - Thanks for the explanation. When an archival preference is known, we definitely like to follow it, so let's have the update merged in.

**Pull request merged**